components/camera/replaypcd/replaypcd_utils_test.go

```diff
@@ -46,6 +46,8 @@ type mockDataServiceServer struct {
 func (mDServer *mockDataServiceServer) BinaryDataByIDs(ctx context.Context, req *datapb.BinaryDataByIDsRequest,
 ) (*datapb.BinaryDataByIDsResponse, error) {
 	// Parse request
+	// adding nolint until the corresponding test is fixed to match the newest API version
```

Comment:
Super minor thing, but please mention the actual jira ticket as a TODO

ex: `TODO(RSDK-3795): Update BinaryDataByIDs test to match newest API`